src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -315,9 +283,11 @@ protected override void Free()
             BoundNode node,
             Binder binder,
             Conversions conversions,
-            ArrayBuilder<(BoundReturnStatement, TypeWithAnnotations)> returnTypesOpt,
             VariableState initialState,
-            Dictionary<BoundExpression, (NullabilityInfo, TypeSymbol)> analyzedNullabilityMapOpt)
+            ImmutableArray<(BoundReturnStatement, TypeWithAnnotations)> returnTypesOpt,
```

Comment:
>ImmutableArray<(BoundReturnStatement, TypeWithAnnotations)> returnTypesOpt [](start = 12, length = 74)

What is the point in passing this array in? Would we ever use its content during analysis?